### PR TITLE
Update 20-default.list

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Please add the following dependency to the rosdep database.

## Package name:

TODO

## Package Upstream Source:

TODO link to source repository

## Purpose of using this:

TODO Replace. This dependency is being used for this reason. This is why I think it's valuable to be added to the rosdep database. 

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here: 

http://sourcecode_url

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: rosdep/sources.list.d/20-default.list [hunk 1]
```diff
@@ -1,10 +1 @@
-# os-specific listings first
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/osx-homebrew.yaml osx
-
-# generic
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/base.yaml
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/python.yaml
-yaml https://raw.githubusercontent.com/ros/rosdistro/master/rosdep/ruby.yaml
-gbpdistro https://raw.githubusercontent.com/ros/rosdistro/master/releases/fuerte.yaml fuerte
```
Comment (wolfv) [inline rosdep/sources.list.d/20-default.list:8]: do we not need the `yaml ...` in front of the filename?

**Comment by wolfv** [comment]
This is exciting :)

Maybe ... at some point ... we should move the robostack.yaml file into a distro-independent location so that it works for foxy and noetic. But I am totally fine with this as it is now!

File: rosdep/sources.list.d/20-default.list (hunk 1 above)
Comment (Tobias-Fischer) [reply rosdep/sources.list.d/20-default.list:8]: We do. Thanks for catching this! It doesn't work yet till merging https://github.com/RoboStack/ros-noetic/pull/119 by the way.

**Comment by Tobias-Fischer** [comment]
Yeah - this is tricky though. How can we handle differing versions of e.g. Gazebo etc.? We might need some older versions in case of noetic sometimes. We should do a diff between the yaml's and see how much they actually differ.

**Pull request merged**